### ISSUE TITLE
Scrolling

### ISSUE THREAD

**Issue by vv73** [mudrd8mz/moodle-editor_marklar#12]
Marklar remembers position of scrolling both in editable and preview mode, but they are different.
Is it possible to synchromize the positions? It is useful for edition long articles.

**Comment by mudrd8mz** [comment]
Yeah, that would be nice to have. Patches welcome (I am not going to work on this myself).
